### ISSUE TITLE
Video caption maker: Not possible to reach items in caption list

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#207]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/captionmaker/

Load sample video, add a caption and save, then try to reach the captions in the right hand area

Expected:

It is possible to click them with the mouse so it should be possible with keyboard

Actual:

They're just divs so not reachable. 

(also in general the buttons and text fields have subtle focus styles)

**Comment by alrra** [comment]
@dstorey @sarvaje This demo is external, namely, it's from https://testdrive-archive.azurewebsites.net/graphics/captionmaker/.

**Comment by dstorey** [comment]
@alrra, @molant should know how to get at the code.
